StudioCore/ParamEditor/ParamEditorScreen.cs

```diff
@@ -450,17 +450,45 @@ public override void DrawEditorMenu()
                     }
                     ImGui.EndMenu();
                 }
-                if (ImGui.MenuItem("Sort rows by ID", _activeView._selection.paramSelectionExists()))
+                if (ImGui.MenuItem("Sort rows by ID", _activeView._selection.activeParamExists()))
                 {
                     EditorActionManager.ExecuteAction(MassParamEditOther.SortRows(ParamBank.PrimaryBank, _activeView._selection.getActiveParam()));
                 }
-                if (ImGui.MenuItem("Import Row Names", "", false, ParamBank.PrimaryBank.Params != null))
+                if (ImGui.BeginMenu("Import Row Names"))
                 {
-                    try {
-                        EditorActionManager.ExecuteAction(ParamBank.PrimaryBank.LoadParamDefaultNames());
-                    } catch {
+                    void ImportRowNames(bool currentParamOnly, string title)
+                    {
+                        const string importRowQuestion = $"Would you like to replace row names with default names defined within DSMapStudio?\n\nSelect \"Yes\" to replace all names, \"No\" to only replace empty names, \"Cancel\" to abort.";
+                        string currentParam = currentParamOnly ? _activeView._selection.getActiveParam() : null;
+                        DialogResult question = MessageBox.Show(importRowQuestion, title, MessageBoxButtons.YesNoCancel, MessageBoxIcon.Question);
```

Comment:
Is it a conscious choice to use messagebox here? Could alternatively put a toggle in the menu. Can't say if there's much downside other than scaring the occasional nighttime user. Maybe if we ever try bouncing from the windows exclusivity it'll mean less work, but that's hardly a large issue.

Reply:
It's a conscious choice because I consider it an invasive action which should come with a warning. I used the same method for displaying this warning that the Upgrade Params function uses.